### PR TITLE
[dualtor][ipv6] Fix `test_orchagent_active_tor_downstream`

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Fix `test_orchagent_active_tor_downstream`.

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

#### How did you do it?
Restore the ipv6 neighbor after removal.

#### How did you verify/test it?
```
dualtor/test_orchagent_active_tor_downstream.py::test_active_tor_remove_neighbor_downstream_active[ipv6] PASSED                                                                                                                                                        [ 50%]
dualtor/test_orchagent_active_tor_downstream.py::test_downstream_ecmp_nexthops[ipv6] PASSED                                                                                                                                                                            [100%]

================================================================================================================= 2 passed, 2 deselected in 1492.69 seconds ==================================================================================================================
```

### CODE REVIEW

File: tests/dualtor/test_orchagent_active_tor_downstream.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def remove_neighbor(ptfhost, duthost, server_ip, ip_version):
             restore = True
             neighbor_advertise_process = "garp_service"
         elif ip_version == "ipv6":
-            restore = False
+            restore = True
```
Comment (yutongzhang-microsoft) [inline tests/dualtor/test_orchagent_active_tor_downstream.py:76]: Is ipv6 neighbor statically configured similar with ipv4 neighbor?

Comment (lolyu) [reply tests/dualtor/test_orchagent_active_tor_downstream.py:76]: Yes, pls check: https://github.com/Azure/sonic-mgmt/blob/1193c9925ba928b991375f3ffabc68562edee5f6/tests/common/dualtor/dual_tor_mock.py#L280

**Review by ZhaohuiS — APPROVED** [review]
LGTM

**Pull request closed without merging**